docs/application/tizen-studio/setup/hardware-accelerated-execution-manager.md

```diff
@@ -97,15 +88,15 @@ If you meet an installation failure because of hardware requirements, check the
 <a name="on_MacOS"></a>
 ## Installing Intel&reg; HAXM on macOS
 
-Supported macOS versions: 10.10 Yosemite (64-bit), 10.9 Mavericks (64-bit), 10.8 Mountain Lion (64-bit)
+Supported macOS versions: 10.12 Sierra (64-bit), 10.11 OS X El Capitan(64-bit), 10.10 Yosemite (64-bit), 10.9 Mavericks (64-bit), 10.8 Mountain Lion (64-bit)
 
-**HAXM is installed automatically as part of the Tizen Studio.** If you want to install the HAXM individually, visit [http://download.tizen.org/sdk/haxm/6.0.3/mac/](http://download.tizen.org/sdk/haxm/6.0.3/mac/) and download the macOS installer package.
+**HAXM is installed automatically as part of the Tizen Studio.** If you want to install the HAXM individually, visit [http://download.tizen.org/sdk/haxm/7.0.0/mac/](http://download.tizen.org/sdk/haxm/7.0.0/mac/) and download the macOS installer package.
 
 **Table: Windows installer package**
 
-| macOS installer                          | File description | File size | MD5 checksum                     |
+| macOS installer                          | File description | File size | SHA256SUM                     |
 |----------------------------------------|----------------|---------|--------------------------------|
-| [IntelHAXM_6.0.3.dmg](http://download.tizen.org/sdk/haxm/6.0.3/mac/IntelHAXM_6.0.3.dmg) | System driver    | 213 KB    | cbd7e3cbdb8b4a84c91fc9094b2cd134 |
+| [IntelHAXM_7.0.0.dmg](http://download.tizen.org/sdk/haxm/7.0.0/mac/IntelHAXM_7.0.0.dmg) | System driver    | 218 KB    | fe7e4cd86b7a2b85591397fd6bf2ef43e7b088bfc72badab0c4d3532a65c2a2f |
```

Comment:
In my opinion there is no point in giving the same link twice in the same section.
Please remove the URL from the table and keep the filename in BOLD.
You can give the full path to the URL in the previous paragraph.